### ISSUE TITLE
Search is performed even no character set

### ISSUE THREAD

**Issue by onetliviunicu** [appirio-tech/topcoder-app-r#170]
Environment

Mozilla Firefox 45.0.1
Chrome 49.0.2623.112
Windows 7

Description

1) Login into application https://www.topcoder-qa.com/search/members/?q=javascript

2) Press on the search icon glass from the top right of page.

3) Set focus on search field but not enter any character.

4) Press enter.

actual: the search is performed and the page with "Oops! There was an error." is returned.

expected: do not allow to be performed the search when no characters are set in field(set a validation on search field!).

see attach!

Notes

- the zip archived contain a video file, please unzip!
[search.zip](https://github.com/appirio-tech/topcoder-app-r/files/211440/search.zip)

**Comment by onetliviunicu** [comment]
This issue can be simulated only with Chrome web browser!

**Comment by callmekatootie** [comment]
I am surprised that this was reported only after the past 169 bugs!! Valid bug

**Comment by nlitwin** [comment]
Might be duplicate of https://github.com/appirio-tech/topcoder-app-r/issues/22
